components/script/script_thread.rs

```diff
@@ -3429,7 +3459,18 @@ impl ScriptThread {
                         let pixel_dist =
                             (pixel_dist.x * pixel_dist.x + pixel_dist.y * pixel_dist.y).sqrt();
                         if pixel_dist < 10.0 * document.window().device_pixel_ratio().get() {
-                            document.note_pending_input_event(event.clone());
+                            // Pass webdriver_id to the newly generated click event
+                            document.note_pending_input_event(ConstellationInputEvent {
```

Comment:
Hmm, the underlying assumption is that each webdriver action is owned by by script thread event, and we are essentially passing the ownership here. It might be a correct assumption though, as we know which one will be handled last.